vlasovsolver/cpu_trans_pencils.cpp

```diff
@@ -0,0 +1,1170 @@
+// use DCCRG version Nov 8th 2018 01482cfba8
+#include "../grid.h"
+using namespace std;
+using namespace spatial_cell;
+
+#include "cpu_trans_pencils.hpp"
+
+std::array<setOfPencils,3> DimensionPencils;
+std::array<std::unordered_set<CellID>,3> DimensionTargetCells;
+
+//Is cell translated? It is not translated if DO_NO_COMPUTE or if it is sysboundary cell and not in first sysboundarylayer
+bool do_translate_cell(SpatialCell* SC){
+   if(SC->sysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE ||
+      (SC->sysBoundaryLayer != 1 && SC->sysBoundaryFlag != sysboundarytype::NOT_SYSBOUNDARY))
+      return false;
+   else
+      return true;
+}
```

Comment:
This wants some PADDINGton? :teddy_bear:

Reply:
no, as Padding type is only for FSgrid cells.